rakelib/user_email_identifier.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'roo'
+
+desc 'Build a CSV of user email identifiers'
+
+namespace :user_email_identifier do
```

Comment:
Can you create a spec for this? It's less important than others since this just gets data instead of writing data, but we should still be testing our rake tasks with specs